packages/data-explorer-ui/src/components/Layout/components/Header/components/ProfileComponent/components/AuthenticationMenu/authenticationMenu.stories.tsx

```diff
@@ -15,7 +15,7 @@ export default {
   title: "Components/Navigation/AuthenticationMenu",
 } satisfies Meta<typeof AuthenticationMenu>;
 
-const AuthenticationMenuTemplate: StoryFn<typeof AuthenticationMenu> = (
+const AuthenticationMenuTemplate: ComponentStory<typeof AuthenticationMenu> = (
```

Comment:
Hi @imadx I think let's revert this - and keep the change requests focused to the ticket - we can resolve this in a new ticket if needed? Thoughts?

Reply:
Reverted this. Not required with the rebased updates of main

Reply:
Hi @imadx, I'm not sure the code is reverted / rebased? I'm still seeing change requests for the story?